src/api/card.ts

```diff
@@ -73,7 +73,7 @@ export const validateAnswer = async (
 ): Promise<FlashcardAnswerValidation> => {
   const response = await fetch(`/api/cards/getCardById/${id}`)
   if (!response.ok) {
-    throw new Error(`HTTP status: ${response.status} - Failed to fetch card `)
+    throw new Error(`HTTP status: ${response.status} - Failed to fetch card.`)
```

Comment:
Merkst hier selber, dass du für eine simple Anpassung den gleichen Error 7x anpassen musstest, das sollten wir generisch behandeln

Reply:
Dokumentiert in #47